tools/Unix.mk

```diff
@@ -27,8 +27,10 @@ endif
 # Build any necessary tools needed early in the build.
 # incdir - Is needed immediately by all Make.defs file.
 
+ifeq ($(wildcard $(TOPDIR)/tools/incdir),)
 DUMMY  := ${shell $(MAKE) -C tools -f Makefile.host incdir \
```

Comment:
can we move the command to target action?

Reply:
sure, let me give it a try